nodejs/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@google-cloud/rcloadenv",
+  "version": "0.1.0",
+  "license": "Apache-2.0",
+  "author": "Google Inc.",
+  "engines": {
+    "node": ">=4"
+  },
+  "bin": {
+    "rcloadenv": "./bin/rcloadenv"
+  },
+  "main": "src/index.js",
+  "scripts": {
+    "lint": "semistandard"
+  },
+  "dependencies": {
```

Comment:
I'm getting circle set up, and I noticed semistandard isn't listed anywhere. Should we add a devDependencies section?